cdist/conf/type/__firewalld_start/gencode-remote

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh -e
+"#!/bin/sh -2
```

Comment:
There seems to be something I am missing here. Why does the line begin with a double-quote and what does `-2` means%

Reply:
@uqam-fob It is a vim typo. I fixed it.